src/ucp/wireup/wireup_cm.c

```diff
@@ -187,6 +187,8 @@ static ssize_t ucp_cm_client_priv_pack_cb(void *arg, const char *dev_name,
     sa_data->ep_ptr    = (uintptr_t)ep;
     sa_data->err_mode  = ucp_ep_config(ep)->key.err_mode;
     sa_data->addr_mode = UCP_WIREUP_SA_DATA_CM_ADDR;
+    sa_data->dev_index = ucp_context_dev_index(worker->context, dev_name);
```

Comment:
ucp_cm_ep_client_initial_config_get already searches for resource by device name
let's not do it again here